modules/builds-strategy-docker-squash-layers.adoc

```diff
@@ -1,18 +1,20 @@
+// Module included in the following assemblies:
+//* builds/running-entitled-builds.adoc
+
 [id="builds-strategy-docker-squash-layers_{context}"]
 = Squash layers with docker builds
 
-Docker builds normally create a layer representing each instruction in a 
-*_Dockerfile_*. Setting the `imageOptimizationPolicy` to `SkipLayers` will merge 
+Docker builds normally create a layer representing each instruction in a
+`Dockerfile`. Setting the `imageOptimizationPolicy` to `SkipLayers` will merge
 all instructions into a single layer on top of the base image.
 
 .Procedure
 
-To set the `imageOptimizationPolicy` to `SkipLayers`:
+* Set the `imageOptimizationPolicy` to `SkipLayers`:
 [source, yaml]
 ----
 strategy:
   dockerStrategy:
     imageOptimizationPolicy: SkipLayers <1>
 ----
-<1> Layers are always squashed in OpenShift 4.0 at present, but this is subject to change 
-in a future release.
+<1> Layers are always squashed in {product-title} {product-version}.
```

Comment:
@bmcelvee Note that in 4.2 this won't be the case. This landed with https://github.com/openshift/builder/pull/35